sdk/nodejs/apigee/endpointAttachment.ts

```diff
@@ -14,39 +14,6 @@ import * as utilities from "../utilities";
  *     * [Creating an environment](https://cloud.google.com/apigee/docs/api-platform/get-started/create-environment)
  *
  * ## Example Usage
- * ### Apigee Endpoint Attachment Basic
```

Comment:
Usually example regressions are due to bugs/limitations in tf2pulumi that fail to convert examples. I don't see a corresponding docs change, so it may be related to the updated pulumi dependencies.

Looking at my local build logs, I see the following error:
```
The example will be dropped from any generated docs or SDKs.
warning: unable to convert HCL example 'Apigee Endpoint Attachment Basic' for Pulumi entity '#/resources/gcp:apigee/endpointAttachment:EndpointAttachment': 1 error occurred:
	* failed to convert HCL for #/resources/gcp:apigee/endpointAttachment:EndpointAttachment to csharp: panic converting HCL: runtime error: invalid memory address or nil pointer dereference
```